pep-0514.txt

```diff
@@ -13,8 +13,8 @@ Abstract
 ========
 
 This PEP defines a schema for the Python registry key to allow third-party
-installers to register their installation, and to allow applications to detect
-and correctly display all Python environments on a user's machine. No
+installers to register their installation, and to allow tools and applications
+to detect and correctly display all Python environments on a user's machine. No
 implementation changes to Python are proposed with this PEP.
 
```

Comment:
There's a few screenshots of Visual Studio [here](https://github.com/Microsoft/PTVS/wiki/Python-Environments), a sample of some of the extra info being shown [here](https://github.com/Microsoft/PTVS/pull/1437). We use everything in the PEP, since I fleshed out the PEP based on what we wanted to use (mostly - there are a few things we didn't bother with, such as localized text). The other IDEs have similar functionality, I just don't have handy screenshots of it..

FWIW, >90% of Windows users of Python don't seem to be using the command line ever...